numpy/distutils/misc_util.py

```diff
@@ -2326,6 +2326,41 @@ def get_info(name):
                 return g.get(name, g.get(name + "_info", {}))
 
             def show():
+                """
+                Print information about various resources (libraries, library
+                directories, include directories, etc.) in the system on which
+                NumPy was built.
```

Comment:
Not sure if we care, but docstrings are supposed to start with a one-like description followed by a blank line, then an extended description.

If you span multiple lines, Sphinx will show only the first one in some places.

Reply:
Good point. I copied this from the old PR, and only made a few edits.  In my local build of the docs, the complete sentence was displayed to the right of the name in the entry that is created in `routines.other.html`, similar to how `deprecate` is displayed at https://numpy.org/devdocs/reference/routines.other.html.

I'll do some more copy-editing and get this down to one line.

Reply:
I might be thinking of a different problem, where Sphinx can't deal with a `.` in the first line.

Reply:
In the latest commit, I added the single line "Print information about BLAS and LAPACK libraries in use." in front of what was already there.